pyEpiabm/pyEpiabm/routine/file_population_config.py

```diff
@@ -150,7 +162,7 @@ def make_pop(input_file: str, random_seed: int = None, time: float = 0):
         return new_pop
 
     @staticmethod
-    def find_cell(population: Population, cell_id: float, current_cell: Cell):
+    def find_cell(population: Population, cell_id: str, current_cell: Cell):
         """Returns cell with given ID in population, creates one if
```

Comment:
Old documentation refers to 'ID', whereas new comments use 'id'. I could be convinced by the new method, as it matches the parameter name, but we need to be consistent.

Reply:
Ah I see, yes will change these